src/dotnet-interactive-vscode/README.md

```diff
@@ -1,21 +1,17 @@
-👉👉👉 This extension is still **under development**.
-
-👉👉👉 Latest **VS Code Insiders** is required and at times this extension might be broken.
+This extension is currently **_in preview_**.  [Visual Studio Code Insiders](https://code.visualstudio.com/insiders/)  is required.
 
 ---
 
 # .NET Interactive Notebooks
 
-This extension adds support for using the `dotnet-interactive` global tool from within VS Code in a notebook-like environment.
+This extension adds support for using .NET Interactive in a Visual Studio Code notebook.
 
 ## Getting Started
 
```

Comment:
1.  Install the latest VS Code Insiders
2.  Install the .NET Interactive Notebook extension from the market place. https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode
3.  Create a new folder `notebooks` and navigate to it
4.  Launch VS Code Insiders like this `code-insiders --enable-proposed-api ms-dotnettools.dotnet-interactive-vscode .`
![image](https://user-images.githubusercontent.com/2546640/82256856-10a04500-9925-11ea-91fd-2ded7c8e10ef.png)
5. Inside notebooks create `foo.dotnet-interactive` and you all set 
![image](https://user-images.githubusercontent.com/2546640/82257512-2cf0b180-9926-11ea-9d33-820c97fca236.png)

Or 
To edit an existing Jupyter notebook, open a file with a `.ipynb` extension,

Reply:
I'd say `Launch VS Code Insiders like this` in `Launch VS Code Insiders in the current directory with the command`